.storybook/recipes/TableCard/TableCard.module.css

```diff
@@ -0,0 +1,23 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #STANDARDSCOVERAGE
+\*------------------------------------*/
+
+/**
+ * 1) 
+ */
+.standards-coverage {
+}
```

Comment:
Empty block, not sure if intentional.
On a side note, I think the stylelint is supposed to [catch empty blocks](https://stylelint.io/user-guide/rules/list/block-no-empty/), not sure if that's running properly @dierat

Reply:
Weird, I'm sure I've seen that error when working in components 🤔

Reply:
Also forgot to change this name to table-card instead of standards coverage. Good catch. Thanks!

Reply:
Tested it in `.storybook/recipes/CardWithNotification/CardWithNotification.module.css` and I'm seeing the error in VSCode at least.
<img width="582" alt="Screen Shot 2022-06-13 at 3 31 06 PM" src="https://user-images.githubusercontent.com/7761701/173458420-3fa88103-553e-4d24-9e34-99503eef86b4.png">

Reply:
I added that empty block to a that recipe and also a component, and only the component showed up when running `yarn lint`.

Reply:
Is it only running in `src`? https://github.com/chanzuckerberg/edu-design-system/blob/c133f36aaa659635b7156dc4b370616dae6be070/package.json#L42